aip/client-libraries/4232.md

```diff
@@ -97,6 +97,14 @@ appear before any optional ones. Code generators implementing this feature
 field after an optional one, and **must** do so if the resulting client library
 would not be valid.
 
+Removing or altering an existing `google.api.method_signature` entry is a
```

Comment:
1. Add `**Important:**` as a prefix.
2. Add that _reordering_ them is also a breaking change.

Reply:
Actually, rather than a callout box, add a `### Compatibility` header. I bet we will want to directly link to this.

Reply:
Great idea. Done.